ovos_plugin_manager/templates/tokenization.py

```diff
@@ -0,0 +1,24 @@
+from mycroft_bus_client.message import dig_for_message
+from quebra_frases import span_indexed_word_tokenize, word_tokenize
+
+
+class Tokenizer:
+    def __init__(self, config=None):
+        self.config = config or {}
+
+    @property
+    def lang(self):
+        lang = self.config.get("lang")
+        msg = dig_for_message()
+        if msg:
+            lang = msg.data.get("lang")
+        return lang or "en-us"
+
+    def span_tokenize(self, text, lang=None):
```

Comment:
Consider docstrings to document the difference between `tokenize` and `span_tokenize`